packages/generate-loader/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "webpack-loader-generator",
+    "version":"2.0.12",
+    "description": "CLI for webpack & friends",
+    "license": "MIT",
+    "private": true,
+    "engines": {
+      "node": ">=6.11.5"
+    },
+    "dependencies": {      
+      "webpack-cli-generators": "2.0.12",
+      "yeoman-environment": "^2.0.0"
```

Comment:
What do you mean here?

Reply:
@dhruvdutt I don't think it's possible. I totally understand your point and it would great to just change the version in one single place and that's it!

@sendilkumarn It means that if we have to upgrade the `yeoman-environment` we have to it everywhere (and we have to remember to do it). @dhruvdutt is suggesting if there's a way to update just in one place so we avoid problems (or less problems). I don't know much about how lerna works, maybe there's a solution for that

Reply:
That is what lerna exactly tries to do, may be that is the whole reason to use something like lerna. (add)[https://github.com/lerna/lerna#add]

Reply:
I don't think we use `yeoman` in all packages